classes/cache/CacheApc.php

```diff
@@ -31,11 +31,16 @@
  */
 class CacheApcCore extends Cache
 {
+    public $apcu;
+
     public function __construct()
     {
-        if (!function_exists('apc_exists')) {
```

Comment:
Whoa! Working overtime? :stuck_out_tongue_winking_eye: 

I did some research on why `function_exists` was used and found this: http://stackoverflow.com/questions/9125904/apc-exist-does-not-exist

Apparently `function_exists` was chosen because older versions of APC do not support it.